docs/index.rst

```diff
@@ -219,6 +219,22 @@ description of the options.
 
 .. _certifi: http://certifi.io/
 
+Accessing over Reverse Proxy:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you have a reverse proxy like nginx configured to protect your elastic server with basic http authentication, then remember the following :
+
++ if you have exposed your elastic search over http://example.com/examplepath, then ::
+
+    es = Elasticsearch(
+        ['http://example.com:80/examplepath'],
+        connection_class=RequestsHttpConnection,
```

Comment:
could I ask why you are changing the class here for `RequestsHttpConnection`? It is not required for this to work and is not recommended for simple use cases where the default `urllib3`-based connection works much better. Thanks!

Reply:
I dont remember now ( after 2 years) why `RequestsHttpConnection` was used instead of the default transport.

1. I might have faced issue getting the default `connection_class` transport to use `http_auth` in the format I have specified. 
2. or maybe for the sake of getting logs from the transport (the default might not have provided the logs)

Going forward: If the default one works well, then there is no need to change `RequestsHttpConnection`